elevation_mapping/src/ElevationMapping.cpp

```diff
@@ -128,8 +133,12 @@ bool ElevationMapping::readParameters()
 
   double minUpdateRate;
   nodeHandle_.param("min_update_rate", minUpdateRate, 2.0);
-  maxNoUpdateDuration_.fromSec(1.0 / minUpdateRate);
-  ROS_ASSERT(!maxNoUpdateDuration_.isZero());
+  if (minUpdateRate == 0.0) {
+    maxNoUpdateDuration_.fromSec(0.0);
+    ROS_INFO("Rate for publishing the map is zero.");
```

Comment:
This info is not correct, you are not just disabling the publishing of the fused elevation map but also the prediction step of the robot's motion. This can lead to the problem that the linearity assumption will break and the data will make no sense. Use something like this:

    ROS_WARN("The parameter `min_update_rate` is zero, which can lead to corrupt data.");